sdk/cloudmachine/Azure.Projects.Provisioning/tests/E2ETests.cs

```diff
@@ -3,38 +3,125 @@
 
 #nullable enable
 
+using System;
+using System.Threading;
 using Azure.AI.OpenAI;
+using Azure.Data.AppConfiguration;
+using Azure.Messaging.ServiceBus;
 using Azure.Projects.KeyVault;
+using Azure.Projects.Ofx;
 using Azure.Projects.OpenAI;
+using Azure.Projects.Storage;
 using Azure.Security.KeyVault.Secrets;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
 using NUnit.Framework;
 using OpenAI.Chat;
+using OpenAI.Embeddings;
 
 namespace Azure.Projects.Tests;
 
 public class E2ETests
 {
+    private static string projectId = "cm00000000000test";
+
     [TestCase("-bicep")]
     //[TestCase("")]
     public void OpenAI(string arg)
     {
-        ProjectInfrastructure infra = new("cm0a110d2f21084bb");
+        ProjectInfrastructure infra = new(projectId);
         infra.AddFeature(new OpenAIModelFeature("gpt-4o-mini", "2024-07-18"));
         if (infra.TryExecuteCommand([arg])) return;
 
-        ProjectClient project = new();
+        ProjectClient project = new(projectId, default);
         ChatClient chat = project.GetOpenAIChatClient();
     }
 
     [TestCase("-bicep")]
     //[TestCase("")]
-    public void ConnectionsInAppConfig(string arg)
+    public void KeyVault(string arg)
     {
-        ProjectInfrastructure infra = new("cm0a110d2f21084bb");
+        ProjectInfrastructure infra = new(projectId);
         infra.AddFeature(new KeyVaultFeature());
         if (infra.TryExecuteCommand([arg])) return;
 
-        ProjectClient project = new();
-        SecretClient secrets = project.GetKeyVaultSecretsClient();
+        ProjectClient project = new(projectId, default);
+        SecretClient secrets = project.GetSecretClient();
+    }
+
+    [TestCase("-bicep")]
+    //[TestCase("")]
+    public void All (string arg)
+    {
+        ProjectInfrastructure infrastructure = new(projectId);
+        AddAllFeratures(infrastructure);
+
+        if (infrastructure.TryExecuteCommand([arg]))
+            return;
+
+        ProjectClient project = new(projectId, default);
+        SecretClient secrets = project.GetSecretClient();
+        BlobContainerClient defaultContainer = project.GetBlobContainerClient();
+        BlobContainerClient testContainer = project.GetBlobContainerClient("test");
+        ConfigurationClient config = project.GetConfigurationClient();
+        ChatClient chat = project.GetOpenAIChatClient();
+        EmbeddingClient embedding = project.GetOpenAIEmbeddingClient();
+        //AzureOpenAIClient openAIClient = project.GetAzureOpenAIClient();
+        ServiceBusClient sb = project.GetServiceBusClient();
+        ServiceBusSender sender = project.GetServiceBusSender("cm_servicebus_topic_private", "cm_servicebus_subscription_private");
+        //ServiceBusReceiver receiver = project.GetServiceBusReceiver("cm_servicebus_topic_private", "cm_servicebus_subscription_private");
```

Comment:
Are the commented lines needed for later? If not, let's remove.

Reply:
yeah, this is my way of tracking todo  :-)